lib/server.js

```diff
@@ -14,6 +14,10 @@ server.connection({ port: Config.PORT });
 
 server.register([
   require('hapi-bookshelf-serializer'),
+  {
+    register: require('hapi-format-error'),
+    options: { joiStatusCode: 422 }
+  },
   require('./plugins/features/movies')
```

Comment:
Not sure if it directly tells you to do this in the miyagi doc, but generally we try to keep the packages we `require` at the top of the file

Reply:
This was the way the miyagi doc was set up i think